src/views/OrderDetail.vue

```diff
@@ -0,0 +1,148 @@
+          </ion-thumbnail>
+          <ion-label>
+            <h5>BRAND</h5>
+            <h2>Virtual Name</h2>
+            <p>{{ $t("Color") }} : color</p>
+            <p>{{ $t("Size") }} : size</p>
+          </ion-label>
+          <ion-note slot="end" color="warning">15 {{ $t("In Stock") }}</ion-note>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-checkbox slot="start" color="primary"> </ion-checkbox>
+          <ion-label>{{ $t("UnFillable") }} </ion-label>
+          <ion-select value="Not in stock">
+            <ion-select-option>{{ $t("Not in stock") }}</ion-select-option>
+          </ion-select>
+        </ion-item>
+      </ion-card-content>
+    </ion-card>
+   <ion-text>
```

Comment:
Why ion-text is used over here ?

Reply:
The text component is a simple component that can be used to style the text .I will change now.